cpp/src/parquet/encoding_test.cc

```diff
@@ -665,6 +666,33 @@ class EncodingAdHocTyped : public ::testing::Test {
     ::arrow::AssertArraysEqual(*values, *result, /*verbose=*/true);
   }
 
+  void PlainTwice(int seed) {
+    auto values_single = GetValues(seed);
```

Comment:
```suggestion
    auto random_values = GetValues(seed);
```

single looks strange here.